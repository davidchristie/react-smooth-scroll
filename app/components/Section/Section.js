import React, { PropTypes } from 'react'

import styles from './Section.css'

export default class Section extends React.Component {
  render () {
    return (
      <section
        className={styles.section}
        id={this.props.id}
        style={this.props.style}
      >
        {this.props.children}
      </section>
    )
  }
}

Section.propTypes = {
  id: PropTypes.string,
  style: PropTypes.object
}
