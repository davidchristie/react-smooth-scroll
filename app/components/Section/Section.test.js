import { shallow } from 'enzyme'
import React from 'react'

import Section from '../Section'

describe('<Section />', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Section />)
    expect(wrapper.getNodes()).toMatchSnapshot()
  })
})
