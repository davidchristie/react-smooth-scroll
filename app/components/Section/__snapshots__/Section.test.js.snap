// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Section /> should match snapshot 1`] = `
Array [
  <section
    className="section"
    id={undefined}
    style={undefined}
/>,
]
`;
